### ISSUE TITLE
PROJETO AVIATOR

### ISSUE THREAD

**Issue by Acleciosilva** [Acleciosilva/Seguran-a-do-trabalho#2]
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from sklearn.linear_model import LinearRegression

# Simulação de dados (substitua por seus dados reais)
historico_rodada = [
    4.91, 1.08, 12.00, 1.65, 2.37, 2.94, 1.78, 4.09, 1.00, 71.61, 33.20, 5.99,
    1.36, 10.14, 1.10, 15.50, 3.53, 3.44, 1.43, 1.99, 2.39, 2.42, 1.99, 1.84, 1.68
]

# Criar DataFrame
df = pd.DataFrame(historico_rodada, columns=["Multiplicador"])

# Calcular média móvel para identificar padrões
df["Media_Movel"] = df["Multiplicador"].rolling(window=10).mean()

# Remover valores NaN
df.dropna(inplace=True)

# Preparar dados para previsão
X = np.arange(len(df)).reshape(-1, 1)
y = df["Media_Movel"].values

# Criar e treinar modelo de regressão
modelo = LinearRegression()
modelo.fit(X, y)

# Prever os próximos 10 multiplicadores
proximas_rodadas = np.arange(len(df), len(df) + 10).reshape(-1, 1)
previsoes = modelo.predict(proximas_rodadas)

# Exibir previsões
df_previsoes = pd.DataFrame({
    "Rodada Prevista": np.arange(len(df) + 1, len(df) + 11),
    "Multiplicador Esperado": previsoes
})

print("\nPrevisões das Próximas Rodadas:")
print(df_previsoes)

# Plotar gráfico de tendências
plt.figure(figsize=(10, 5))
plt.plot(df["Multiplicador"], label="Multiplicador Real", linestyle="--", alpha=0.7)
plt.plot(df["Media_Movel"], label="Média Móvel (10 rodadas)", linewidth=2)
plt.xlabel("Rodadas")
plt.ylabel("Multiplicador")
plt.title("Análise de Tendência no Aviator")
plt.legend()
plt.grid()
plt.show()
